.circleci/d2tag.sh

```diff
@@ -0,0 +1,80 @@
+set -e
+
+# Performs a GitHub API call
```

Comment:
Why not beaver? As a first hack you could just assign the `TRAVIS_*` variables from the contents of `CIRCLE_` variables. Then if this works some auto-detection can be added to beaver (hopefully temporarily until we move away from Travis, if CircleCI proves to be a better alternative).

Reply:
I have removed beaver submodule completely for now and copying necessary utils from it on per need basis. Reason is that beaver is currently not developed with CircleCI specifics in mind and I don't want to risk something accidentally getting exposes because of seemingly innocent change upstream. When it will be time to decide if we keep or drop CircleCI here, we shall modify turtle/beaver accordingly, but until that point it is more convenient this way.

Reply:
Sounds fair 👍